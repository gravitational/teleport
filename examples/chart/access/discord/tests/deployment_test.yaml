suite: Test deployment
templates:
  - deployment.yaml
tests:
  - it: should match the snapshot
    set:
      image:
        repository: gcr.io/overridden/repository
        tag: v98.76.54
    asserts:
      - matchSnapshot: {}
