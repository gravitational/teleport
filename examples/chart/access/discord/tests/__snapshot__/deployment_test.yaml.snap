should match the snapshot:
  1: |
    apiVersion: apps/v1
    kind: Deployment
    metadata:
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: teleport-plugin-discord
        app.kubernetes.io/version: 17.0.0-dev
        helm.sh/chart: teleport-plugin-discord-17.0.0-dev
      name: RELEASE-NAME-teleport-plugin-discord
    spec:
      replicas: 1
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME
          app.kubernetes.io/name: teleport-plugin-discord
      template:
        metadata:
          labels:
            app.kubernetes.io/instance: RELEASE-NAME
            app.kubernetes.io/managed-by: Helm
            app.kubernetes.io/name: teleport-plugin-discord
            app.kubernetes.io/version: 17.0.0-dev
            helm.sh/chart: teleport-plugin-discord-17.0.0-dev
        spec:
          containers:
          - command:
            - /usr/local/bin/teleport-plugin
            - start
            - --config
            - /etc/teleport-discord.toml
            env:
            - name: TELEPORT_PLUGIN_FAIL_FAST
              value: "true"
            image: gcr.io/overridden/repository:v98.76.54
            imagePullPolicy: IfNotPresent
            name: teleport-plugin-discord
            resources: {}
            securityContext: {}
            volumeMounts:
            - mountPath: /etc/teleport-discord.toml
              name: config
              subPath: teleport-discord.toml
            - mountPath: /var/lib/teleport/plugins/discord/teleport-identity
              name: teleport-identity
            - mountPath: /var/lib/teleport/plugins/discord/discord-token
              name: password-file
              subPath: discordToken
          securityContext: {}
          volumes:
          - configMap:
              defaultMode: 384
              name: RELEASE-NAME-teleport-plugin-discord
            name: config
          - name: teleport-identity
            secret:
              defaultMode: 384
              secretName: ""
          - name: password-file
            secret:
              defaultMode: 384
              secretName: RELEASE-NAME-teleport-plugin-discord-secret
